src/main/java/ooo/oshi/foreign/windows/WindowsLibrary.java

```diff
@@ -0,0 +1,32 @@
+package ooo.oshi.foreign.windows;
+
+import java.lang.foreign.FunctionDescriptor;
+import java.lang.foreign.Linker;
+import java.lang.foreign.SymbolLookup;
+import java.lang.invoke.MethodHandle;
+
+import static java.lang.foreign.ValueLayout.JAVA_INT;
+
+public class WindowsLibrary {
```

Comment:
The reason I chose `SystemLibrary` on macOS is because it's actually mapping from the `System` framework.  Not sure what DLL the windows "natives" come from... it looks like it's a C Runtime (CRT) function and there are restrictions on portability of those.

So great job getting this working, but per my comment on the outside thread, you probably want to stick to Win32 API functions.